test/odbc/CMakeLists.txt

```diff
@@ -23,18 +23,15 @@ add_executable(
 
   main
```

Comment:
Could we do something like this
https://github.com/Bit-Quill/babelfish_extensions/pull/10/files#diff-da76a97fb2ac97882e868bf5b20711a1fca7048fcbb8e6400731a569fcd7cce2

Separating MSSQL & PSQL

Reply:
Good call. Done.

I've also moved all the helper files (non-test files) such as `odbc_handler.h/cpp`, `query_generator.h/cpp`, etc. into a new directory called `src/` for better organization. I called this folder `src` to align with the naming scheme used in the `JDBC/` & `dotnet/` test frameworks.